WolvenKit.App/ViewModels/Tools/ChunkViewModelExtended/ChunkViewModel.ContextMenu.cs

```diff
@@ -99,6 +99,11 @@ private async Task RenameMaterial()
         var oldName = entry.Name;
         var newName = await Interactions.ShowInputBoxAsync("Rename material", oldName.GetResolvedText() ?? "");
 
+        if (string.IsNullOrEmpty(newName.Trim()))
```

Comment:
Use IsNullOrWhiteSpace here instead of Trim: `string.IsNullOrWhiteSpace(newName)`